index.html

```diff
@@ -0,0 +1,214 @@
+                            <img class="likeBarElement"  src="images/shareButton.png" alt="Click to share">                    
+                        </button>   
+                        <button class="buttonAll" id="addButton">
+                            <img class="likeBarElement"  src="images/addListButton.png" alt="Click to save">                
+                        </button>   
+                        <button class="buttonAll" id="threeDotButton">
+                            <img class="likeBarElement"  src="images/threeDotButton.png" alt="Click to see more options">                            
+                        </button>   
+                        <button class="buttonAll" id="likeButton">
+                            <img class="likeBarElement"  src="images/likeButton.png" alt="Click to like">
+                        </button>  
+                        <button class="buttonAll" id="unlikeButton">
+                            <img class="likeBarElement"  src="images/unlikeButton.png" alt="Click to unlike">
+                        </button>                      
+                    </div>
+                </div>
+                <br>
+                <div class="description">
+                    <h2>Published on Feb 27,2014</h2>
+                    <p>A befitting tribute to former PM #AtalBihariVajpayee, India’s longest Rail-cum-Road Bridge, #BogibeelBridge over river Brahmaputra is a construction marvel and a milestone for the development of North East. Watch to know more.</p>
+                    <h4 class="more">SHOW MORE</h4>   
+                </div>
+               
+            </section>
+            <aside id="sidebar">
+                <div id="nextBar">
+                    <h1 id="nextText">Up Next</h1>
+                    <h1 id="autoplay">Autoplay</h1>
+                    <img id="autoplayButton" src="sidebar/autoplayButton.png" alt="Click to autoplay">
+                </div>
+                <div id="suggestions">
+                    <div class="suggestionElement"> 
+                        <img class="sugImage" src="sidebar/suggestion1.png" alt="One of suggestion videos" >
+                        <div class="sugProperites">
+                            <h2 class="sugName">Your First GitHub Pull Request (in 10 Mins)</h2>
+                            <h2 class="sugAuthor">Jackson Bates</h2>
+                            <h2 class="sugView">43 K view</h2>
+                        </div>                    
+                    </div>
+                    <div class="suggestionElement"> 
+                        <img class="sugImage" src="sidebar/suggestion2.png" alt="One of suggestion videos" >
+                        <div class="sugProperites">
+                            <h2 class="sugName">Contributing to Open Source Part I: The Easy Way - YouTube</h2>
+                            <h2 class="sugAuthor">The Odin Project</h2>
+                            <h2 class="sugView">33 K view</h2>
```

Comment:
- [ ] Again, do not overuse h tags. [Read more on how to use them here.](https://www.w3schools.com/tags/tag_hn.asp)
In the case of the recommended video only the video title would make semantic sense as an h2.